### DISCUSSION TITLE
How can I establish connections through k8 nginx ingress which don't close

### DISCUSSION THREAD

**Discussion by pnikhil610** [aklivity/zilla#1129] [Q&A]
I've setup Zilla on k8. It works great!

However, I've issue with connections closing at random interval.

![image](https://github.com/aklivity/zilla/assets/115134190/02e808a2-a8dd-4783-80e7-3946d385108b)

Here's how ingress look likes:

![image](https://github.com/aklivity/zilla/assets/115134190/249b5223-974c-4bb7-afa5-3945a5309cd7)

I've enabled http/2 
![2024-07-03_14-20](https://github.com/aklivity/zilla/assets/115134190/aa69f0b2-3fb7-4a20-8827-2f9840724c9a)

Is there any example for ideal k8 nginx ingress setup which I'm missing and is causing this issue?

**Comment by pnikhil610** [comment]
I want to add that `ERR_CONNECTION_CLOSED` is new error I started getting after update to v0.9.83 @vordimous

**Comment by vordimous** [comment]
@pnikhil610 I have found some additional settings that can affect the SSE connections through an nginx-ingress controller. Can you update your annotations to add these:

```
    nginx.ingress.kubernetes.io/proxy-buffering: "off"
    nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
    nginx.ingress.kubernetes.io/proxy-connect-timeout: "21600"
    nginx.ingress.kubernetes.io/proxy-read-timeout: "21600"
    nginx.ingress.kubernetes.io/proxy-send-timeout: "21600"
```

One important thing is to make sure proxy buffering is off so it won't affect the SSE stream itself.
